sdk/src/timing.rs

```diff
@@ -2,11 +2,11 @@
 use std::time::Duration;
 use std::time::{SystemTime, UNIX_EPOCH};
 
-pub const NUM_TICKS_PER_SECOND: u64 = 10;
+pub const NUM_TICKS_PER_SECOND: u64 = 100;
```

Comment:
Big waste of space and network bandwidth?

Reply:
How do we verify PoH in a timely manner then?

Reply:
And at 4K ticks per second, it’s 0.1% of 1gbps.